Tools/clinic/clinic.py

```diff
@@ -1492,7 +1492,7 @@ def render_function(self, clinic, f):
 
         template_dict['c_basename'] = c_basename
 
-        methoddef_name = "{}_METHODDEF".format(c_basename.upper())
+        methoddef_name = f"{c_basename.upper()}_METHODDEF"
         template_dict['methoddef_name'] = methoddef_name
```

Comment:
In this case, I actually find a simple concat easier to read:

```suggestion
        template_dict['methoddef_name'] = c_basename.upper() + "_METHODDEF"
```

Reply:
Hmm, I prefer the f-string myself, but happy to go with a concat :)